### DISCUSSION TITLE
[Question]: How to install custom go2rtc version?

### DISCUSSION THREAD

**Discussion by rocket59** [blakeblackshear/frigate#10200] [Ask A Question] [answered]
### What is your question:

Was just looking through the docs about doing this https://docs.frigate.video/configuration/advanced/#custom-go2rtc-version

Are these the correct files/steps to use?
1. Get the file from https://github.com/AlexxIT/go2rtc/releases
2. So for proxmox on an intel machine grab this one: go2rtc_linux_amd64
3. rename to go2rtc as per docs and follow rest of instructions.

**Comment by NickM-27** [comment, marked as answer]
yes, that is correct

**Reply by rocket59** [reply]
Thanks!
